packages/docs-page/server/index.ts

```diff
@@ -31,6 +31,11 @@ export function getStaticGenerationFunctions(
         localContentDir: string
         navDataFile: string
         strategy: 'fs'
+        /** Optional location of the partials directory
+         * relative to process.cwd().
+         * Passed to our resolveIncludes plugin.
+         * Defaults to "content/partials". */
```

Comment:
Have added an annotation at this level, so that when using server functions in eg. `/docs/[[page]].tsx` files, there's a bit of helpful descriptive text.